test/database/create_storage_tables_user_attachments_migration.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class CreateStorageTablesUserAttachmentsMigration < ActiveRecord::Migration[7.0]
+  def change
+    create_table :storage_tables_user_attachments, force: true do |t|
+      t.string :name,         null: false
+      t.string     :filename, null: false
+      t.belongs_to :record,   null: false, foreign_key: { to_table: :users }
+      t.string :checksum, null: false, foreign_key: false
+      t.string :blob_key, null: false
```

Comment:
Van text kolommen met een fixed width kunnen we beter voor de `char(n)` datatype gaan. Dit bespaart ruimte aangezien Postgresql hier beter mee kan optimaliseren.